database/migrations/2024_06_20_165116_add_rounding_columns_to_price_lists_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('price_lists', function (Blueprint $table) {
+            $table->string('rounding_method_enum')->default('none')->after('price_list_code');
+            $table->integer('rounding_precision')->nullable()->after('rounding_method_enum');
```

Comment:
can this be negative?

Reply:
yes, intended, so that it is possible to round to hundreds, etc.